ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -352,6 +352,7 @@ a database ( directory )
     context.getFsScratchDirs().putAll(loadContext.pathInfo.getFsScratchDirs());
     if (!HiveConf.getBoolVar(conf, REPL_DUMP_SKIP_IMMUTABLE_DATA_COPY)) {
       createReplLoadCompleteAckTask();
+      createNotificationFileTask();
```

Comment:
Shouldn't ack creation be the last thing?

Reply:
Yes, It should be. I'll change.